v2/src/lib.rs

```diff
@@ -640,54 +639,13 @@ impl Storage for InfluxDbStorage {
         };
         let mut result: Vec<StoredData> = vec![];
 
-        for zpoint in query_result {
-            // get the encoding
-            let encoding_prefix =
-                (if zpoint.encoding_prefix >= 0 && zpoint.encoding_prefix <= 255 {
-                    Ok(zpoint.encoding_prefix as u8)
-                } else {
-                    Err(zerror!(
-                        "Encoding {} is outside possible range of values",
-                        zpoint.encoding_prefix
-                    ))
-                })
-                .and_then(|prefix| {
-                    KnownEncoding::try_from(prefix)
-                        .map_err(|_| zerror!("Unknown encoding {}", zpoint.encoding_prefix))
-                })?;
-            let encoding = if zpoint.encoding_suffix.is_empty() {
-                Encoding::Exact(encoding_prefix)
-            } else {
-                Encoding::WithSuffix(encoding_prefix, zpoint.encoding_suffix.into())
-            };
-            // get the payload
-            let payload = if zpoint.base64 {
-                match b64_std_engine.decode(zpoint.value) {
-                    Ok(v) => ZBuf::from(v),
-                    Err(e) => {
-                        warn!(
-                            r#"Failed to decode zenoh base64 Value from Influx point with timestamp="{}": {}"#,
-                            zpoint.timestamp, e
-                        );
-                        continue;
-                    }
-                }
-            } else {
-                ZBuf::from(zpoint.value.into_bytes())
-            };
-            // get the timestamp
-            let timestamp = match Timestamp::from_str(&zpoint.timestamp) {
-                Ok(t) => t,
-                Err(e) => {
-                    warn!(
-                        r#"Failed to decode zenoh Timestamp from Influx point with timestamp="{}": {:?}"#,
-                        zpoint.timestamp, e
-                    );
-                    continue;
-                }
-            };
-            let value = Value::new(payload).encoding(encoding);
-            result.push(StoredData { value, timestamp });
+        for i in &query_result {
```

Comment:
Why this change in conversion of `zpoint` to `StoredData` ?
Here the encoding will be lost, and the value always returned as a `String` (since `ZenohPoint.value` is a `String`). Moreover the case of a `base64` string is no longer treated.

Reply:
Fixed.